test/controllers/welcome_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class WelcomeControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get root_url
```

Comment:
ok that makes sense, I forgot about that change and how the controllers are now really integration tests.